### ISSUE TITLE
Simplify the example.settings.yml file

### ISSUE THREAD

**Issue by philipnorton42** [hashbangcode/vlad#176]
Just a thought, but I was wondering if we could simplify the example.settings.yml file so that it's just some basic options and what apps to install. This would make it slightly less daunting for newbies to the system and would mean we could add variables without having to add them in more than one place.

We could also create an insanely complete settings file on the documentation site just so that everything is in one place if it's needed.

**Comment by danbohea** [comment]
I've been thinking the same thing for a while now but since the docs have gone live along with a complete list of available vars, plus the fact that we now have more complete default var values, I just keep forgetting that we even still have example.settings.yml.

We could add a terrifying, blood sucking, neck biting, comprehensive example settings file to the docs but to me that overlaps with the vars documentation a fair bit. Personally, I'd rather see the vars docs page get some formatting love first (this has been on my own todo list for a while but time continues to elude me ATM).

In short, yes. Let's make it way more minimal - this would also have the added benefit of having to update it less often. Perhaps we should also add a commented out link to the vars docs page at the top?

**Comment by philipnorton42** [comment]
All good suggestions Dan, thanks :)

**Comment by philipnorton42** [comment]
Everything is in place here. Unless we need to remove more stuff from the file, but I think it's pretty minimal. Closing :)
